pyproject.toml

```diff
@@ -27,8 +27,9 @@ kgcl-rdflib = "^0.3.0"
 pystow = "^0.4.4"
 class-resolver = "^0.3.10"
 ontoportal-client = "0.0.3"
-curies = "^0.1.5"
+curies = "^0.3.0"
 bioregistry = "^0.5.64"
+bioontologies = "^0.2.0"
```

Comment:
shouldn't be necessary

Reply:
Yeah, I agree. I'll remove this. I think I only added this here because I was using an interface and not an implementation and having issues because of that.